2_Operations/kanban.md

```diff
@@ -0,0 +1,37 @@
+# Kanban
+
+Our work process is based on [Kanban](https://en.wikipedia.org/wiki/Kanban) with some elements from [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development)):
+
+ * We are our own *product owners*, as we have no customer projects.
+ * We work asynchronously, so keep scheduled meetings to a bare minimum.
+ * We organize our work using a Kanban board.
+
+Read about [writing User Stories and issues](user-stories.md).
+
+User Stories that are dependent on each other should be merged into one bigger user story. This kind of User Stories can have 2 assignees.
+
+## Monday Meetings
+
+Each Monday we have a meeting to review the Kanban board and check in with our [OKR's](https://en.wikipedia.org/wiki/Objectives_and_key_results). We typically have one meeting per project starting with Operations.
+
+Turn on your camera during the meeting. Try to refrain from doing other things (browsing, doing support, etc.) during the meeting -- we only have this meeting and it doesn't take a lot of time so we should all pay attention to everyone else for this period.
+
+## Urgent Production Fixes
+
+If we need to fix an urgent bug, we use the Scrum *priority lane* approach: the user story gets written and then if the product owner decides it really is urgent, the user story is added to the 'In Progress' pipeline and labeled with Priority Lane so that everyone knows it is an exceptional and urgent issue and the whole team needs to focus on getting it completed as soon as possible.
+
```

Comment:
In Kanban, priority lane is not needed. There are only a very limited amount of stories in progress, so any new story is by default immediatelly being worked on. Note that to add a story to WIP, you need to remove one first..

Reply:
I thought this might be the case, removing!